app/mailers/issue_mailer.rb

```diff
@@ -0,0 +1,13 @@
+# Encoding: utf-8
+class IssueMailer < ApplicationMailer
+  default from: 'new_issue@newvasjuky.com'
+
+  def issue_created(issue_id, user_id)
```

Comment:
you can remove user_id from params
you can get @user as `@issue.user` in mailer if needed